src/proto/flwr/proto/task.proto

```diff
@@ -45,3 +46,31 @@ message TaskRes {
   string workload_id = 3;
   Task task = 4;
 }
+
+message SecAggMsg {
+  message Arrays {
+    message Plural { repeated bytes value = 1; }
+    oneof value {
+      bytes singular = 1;
+      Plural plural = 2;
```

Comment:
Could we just use a list of length one for `singular`? That'd allow us to simplify and get rid of the `oneof`.